src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -283,14 +500,15 @@ static void subCacheItemUpdate(const char* tenant, BSONObj* subP)
   //
   // 03. Create CachedSubscription
   //
-  CachedSubscription* cSubP = (CachedSubscription*) calloc(1, sizeof(CachedSubscription));
+  CachedSubscription* cSubP = new CachedSubscription();
 
   if (cSubP == NULL)
   {
-    LM_E(("Runtime Error (cannot allocate memory for a cached subscription: %s)", strerror(errno)));
+    LM_X(1, ("Runtime Error (cannot allocate memory for a cached subscription: %s)", strerror(errno)));
     return;
   }
 
+  // LM_M(("Allocated CachedSubscription at %p", cSubP));
```

Comment:
I don't paying too much attengion to LM_M (either commented or not), understanding we will clean all this in the last steps of the "cache recovery" work.

NTC